lib/tests/streamlit/state/session_state_test.py

```diff
@@ -33,191 +32,191 @@
 
 # TODO: Mock widget values and modify the tests below to account for this once
 #       session_state and widget state are unified.
-class SessionStateTests(unittest.TestCase):
```

Comment:
nit: would prefer to either remove these entirely until they can be rewritten or use `@pytest.mark.skip` to ignore them (the second is probably preferable)